index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio</title>
+</head>
+<body>
+    <header>
+        <div id="welcome">
+            Welcome
+        </div>
+        <Nav id="menu">
+        <img src="./Images/icon-menu.png" alt="Menu Icon">
+        </Nav>
+    </header>
+<section class="bg">
+<div class="highlight">
+    Hey there. I’m Usama I’m a software developer
+</div>
+<div class="intro">
+    I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+    don’t hestiate to contact me.
+</div>
+     <nav class="social-media">
+     <img src="./Images/social-media-icons/twitter.svg" alt="Link to Twitter">
+     <img src="./Images/social-media-icons/Linkedin.svg" alt="Link to Linkedin">
+     <img src="./Images/social-media-icons/medium.svg" alt="Link to Medium">
+     <img src="./Images/social-media-icons/github.svg" alt="Link to Github">
+     <img src="./Images/social-media-icons/angellist.svg" alt="Link to Angellist">
+    </nav>
```

Comment:
- [ ] Instead of nav tag please use ul and li tags for managing your images because of SEO purpose :+1:
- [ ] Please make all your each image inside an `a` tag because these icons are links to your social accounts :blush: